### PR TITLE
List: organization owners (v2)

### PR DESCRIPTION
**Pull request by larsxschneider**
Based on @mlbright and @pluehne 's work in #36 I finished the org owners feature. If you have time, please give it a look.

### CODE REVIEW

File: updater/reports/Report.py [hunk 1]
```diff
@@ -100,6 +100,15 @@ def executeScript(self, script, stdin = None):
 
 		return stdout
 
+	def executeGHEConsole(self, rubyCode):
```
Comment (mlbright) [inline updater/reports/Report.py]: 🥇

**Comment by pluehne** [comment]
@larsxschneider: I’ll try to have a look at this tomorrow 😃.

**Comment by mlbright** [comment]
@larsxschneider, @pluehne feel free to close #36

**Comment by pluehne** [comment]
@mlbright: It was already closed by @larsxschneider earlier.

**Pull request merged**